relay-event-schema/src/protocol/event.rs

```diff
@@ -805,6 +807,15 @@ impl Getter for Event {
             }
         })
     }
+
+    fn get_iter(&self, path: &str) -> Option<GetterIter<'_>> {
+        Some(match path.strip_prefix("event.")? {
+            "exceptions.values" => {
```

Comment:
In the protocol, the field is called `exception` ([here](https://github.com/getsentry/relay/blob/ec7ad6bbefb9f14716819e731c15d461e0b935ac/relay-event-schema/src/protocol/event.rs#L381)).

```suggestion
            "exception.values" => {
```